### PR TITLE
Get config information (related to issue #67)

### PR DESCRIPTION
**Pull request by tgmike**
Related to #67 

Added a method to client that allows for mountebank configuration and process information to be gotten via a config object.

### CODE REVIEW

**Comment by mattherman** [comment]
I'm hoping to review this sometime this week. Apologies for the long turnaround.

File: MbDotNet/Models/Others/Config.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace MbDotNet.Models.Others{
```
Comment (mattherman) [inline MbDotNet/Models/Others/Config.cs:4]: Instead of an `Others` namespace, I would just put this in the base `MbDotNet.Models` namespace rather than attempting to categorize it further.

File: MbDotNet/MountebankClient.cs [hunk 2]
```diff
@@ -276,5 +277,15 @@ public async Task DeleteSavedRequestsAsync(int port, CancellationToken cancellat
...
         }
+
+        /// <summary>
+        /// Gets the configuration information of Mountebank
+        /// </summary>
+        /// <returns>A Config object containing the configuration of Mountebank</returns> 
+        public async Task<Config> GetConfigAsync(CancellationToken cancellationToken = default)
+        {
+            var result = await _requestProxy.GetConfigAsync(cancellationToken);
+            return result;
```
Comment (mattherman) [inline MbDotNet/MountebankClient.cs:288]: Remove the separate `return` statement and make this a single-line function.

File: MbDotNet.Tests/Acceptance/ConfigTests.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            Assert.AreEqual(mock, result.Options["mock"]);
+        }
+
+        [TestMethod]
+        public async Task GetConfig_Process()
+        {
+            const string nodeversion = "v14.17.2";
+            var result = await _client.GetConfigAsync();
+            Assert.AreEqual(nodeversion, result.Process["nodeVersion"]);
+        }
```
Comment (mattherman) [inline MbDotNet.Tests/Acceptance/ConfigTests.cs:46]: These tests will pass/fail based on the configuration of the system that runs them so they are not reliable and should be removed. You can add additional asserts to the remaining test to verify that the `Options` and `Process` collections are not empty.

**Review by mattherman — CHANGES_REQUESTED** [review]

File: MbDotNet.Tests/Acceptance/ConfigTests.cs (hunk 3 above)
Comment (mattherman) [reply MbDotNet.Tests/Acceptance/ConfigTests.cs:46]: You can see an example of this in the failing CI build - 

```
  X GetConfig_Process [19ms]
  Error Message:
   Assert.AreEqual failed. Expected:<v14.17.2>. Actual:<v16.15.1>. 
  Stack Trace:
     at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid3[T0,T1,T2](CallSite site, T0 arg0, T1 arg1, T2 arg2)
   at MbDotNet.Tests.Acceptance.ConfigTests.GetConfig_Process() in /home/runner/work/MbDotNet/MbDotNet/MbDotNet.Tests/Acceptance/ConfigTests.cs:line 45
```

**Comment by mattherman** [comment]
This change has been published in v5.0.0-rc8.
